Package.swift

```diff
@@ -241,6 +242,20 @@ var package = Package(
       .copy("opml-test-files/states.opml"),
     ]),
     .target(name: "CodableHelpers"),
+    .target(
+      name: "BraveFavicon",
```

Comment:
Naming nit: Not a brave feature, maybe better to call it `Favicons` or something similar

Reply:
Fixed